### ISSUE TITLE
manageadmins.py not adding adminstrators

### ISSUE THREAD

**Issue by rhinbourgeois** [meraki/automation-scripts#24]
Executing the command on manageadmins.py appears to work in the command prompt, but doesn't populate in the dashboard. Here is the syntax I'm using:

python manageadmins.py -k 1234 -o ORGNAME -c add -a user@domain.com -n User -p full

I've executed the command without the -p full and the -n with no issues. However, when I check the dashboard nothing updates. Additionally, I tried using the network name and the command ran, but never updated in the dashboard. Any help would be greatly appreciated.

**Comment by achill-es** [comment]
Hi,

did you ever try python3 (version > 3.x)? I've had a similar issue recently and this "trick" (moving from implicit python2 to explicit python3) did it immediately!

Regards.

**Comment by mpapazog** [comment]
You can check your Python version with command "python --version".

**Comment by rhinbourgeois** [comment]
achill-es and mpapazog,

On the PC side it's not working and I have confirmed I am using python 3. The command executes, but does nothing in the console. On the MAC side (Confirmed I'm using python3 by using the python3 line with the full command), the command executes tells me the following:

@ INFO: Creating admin "user@domain.com" in org "NAMEDORG"
@ WARNING: Unable to create admin "user@domain.com" in org "NAMEDORG"

However, when I checked the dashboard the user was listed. I'm having the user check to see if he has the appropriate access, but I thought the warning message was odd.

Lastly, thanks for you support here. Greatly appreciate it.

-Rhin
